pkg/azure/client.go

```diff
@@ -197,6 +197,10 @@ func (az *azClient) UpdateGateway(appGwObj *n.ApplicationGateway) (err error) {
 		return
 	}
 
+	if appGwFuture.PollingURL() != "" {
+		glog.V(3).Infof("OperationID='%s'", GetOperationIDFromPollingURL(appGwFuture.PollingURL()))
```

Comment:
add a warning when getOperationIDFromPollingURL returns ""?

Reply:
it would be an error condition and will exist before we come here.